public/keymaps/k/keebio_quefrency_rev1_default.json

```
@@ -1,22 +1,22 @@
 {
   "keyboard": "keebio/quefrency/rev1",
   "keymap": "default",
-  "commit": "e6b9980bd45c186f7360df68c24b6e05a80c10dc",
-  "layout": "LAYOUT",
+  "commit": "5538bffb8eff5203fdabef8f4d8913999177fa53",
+  "layout": "LAYOUT_65_with_macro",
   "layers": [
     [
-      "KC_GESC",  "KC_1",     "KC_2",     "KC_3",     "KC_4",     "KC_5",     "KC_6",     "KC_7",     "KC_8",     "KC_9",     "KC_0",     "KC_MINS",  "KC_EQL",   "KC_DEL",   "KC_BSPC",
-      "KC_TAB",   "KC_Q",     "KC_W",     "KC_E",     "KC_R",     "KC_T",     "KC_Y",     "KC_U",     "KC_I",     "KC_O",     "KC_P",     "KC_LBRC",  "KC_RBRC",  "KC_BSLS",
-      "KC_CAPS",  "KC_A",     "KC_S",     "KC_D",     "KC_F",     "KC_G",     "KC_H",     "KC_J",     "KC_K",     "KC_L",     "KC_SCLN",  "KC_QUOT",  "KC_ENT",
-      "KC_LSFT",  "KC_Z",     "KC_X",     "KC_C",     "KC_V",     "KC_B",     "KC_N",     "KC_M",     "KC_COMM",  "KC_DOT",   "KC_SLSH",  "KC_RSFT",  "KC_UP",
-      "KC_LCTL",  "KC_LALT",  "KC_LGUI",  "MO(1)",    "KC_SPC",   "MO(1)",    "KC_BSPC",  "KC_RALT",  "KC_RCTL",  "KC_LEFT",  "KC_DOWN",  "KC_RGHT"
+      "KC_F1",    "KC_F2",    "KC_GESC",  "KC_1",     "KC_2",     "KC_3",     "KC_4",     "KC_5",     "KC_6",     "KC_7",     "KC_8",     "KC_9",     "KC_0",     "KC_MINS",  "KC_EQL",   "KC_DEL",   "KC_BSPC",  "KC_HOME",
+      "KC_F3",    "KC_F4",    "KC_TAB",   "KC_Q",     "KC_W",     "KC_E",     "KC_R",     "KC_T",     "KC_Y",     "KC_U",     "KC_I",     "KC_O",     "KC_P",     "KC_LBRC",  "KC_RBRC",  "KC_BSLS",  "KC_END",
+      "KC_F5",    "KC_F6",    "KC_CAPS",  "KC_A",     "KC_S",     "KC_D",     "KC_F",     "KC_G",     "KC_H",     "KC_J",     "KC_K",     "KC_L",     "KC_SCLN",  "KC_QUOT",  "KC_ENT",   "KC_PGUP",
+      "KC_F7",    "KC_F8",    "KC_LSFT",  "KC_Z",     "KC_X",     "KC_C",     "KC_V",     "KC_B",     "KC_N",     "KC_M",     "KC_COMM",  "KC_DOT",   "KC_SLSH",  "KC_RSFT",  "KC_UP",    "KC_PGDN",
+      "KC_F9",    "KC_F10",   "KC_LCTL",  "KC_LALT",  "KC_LGUI",  "MO(1)",    "KC_SPC",   "MO(1)",    "KC_BSPC",  "KC_RALT",  "KC_RCTL",  "KC_RGUI",   "KC_LEFT",  "KC_DOWN",  "KC_RGHT"
```

Matches the `default65macro` keymap on `qmk_firmware`.

```suggestion
      "KC_F9",    "KC_F10",   "KC_LCTL",  "KC_LALT",  "KC_LGUI",  "MO(1)",    "KC_SPC",   "MO(1)",    "KC_SPC",   "KC_RALT",  "KC_RCTL",  "KC_RGUI",   "KC_LEFT",  "KC_DOWN",  "KC_RGHT"
```